src/main/java/org/generationcp/middleware/api/nametype/GermplasmNameTypeService.java

```diff
@@ -24,4 +25,10 @@
 	List<GermplasmNameTypeDTO> filterGermplasmNameTypesByName(String name);
 
 	List<GermplasmNameTypeDTO> getNameTypesByGIDList(List<Integer> gidList);
+
+	boolean existNameTypeUsedInListDataProp(String nameType);
+
+	void updateNameType(GermplasmNameTypeRequestDTO germplasmNameTypeRequestDTO, Integer nameTypeId);
```

Comment:
a minor suggestion. I prefer to set the id first in the list of parameters, that refers to the entoty that it will be modified

Reply:
done.